.circleci/config.yml

```diff
@@ -136,13 +136,10 @@ jobs:
     <<: *default_machine_config
     steps:
       - checkout
+      - setup_node12
```

Comment:
I'm just trying to understand how that config works 🙂So 'setup-node12' is a new step, where is it defined?

Reply:
Check the changes commit by commit, not by overall diff

Reply:
Actually my bad, it's not in this PR

It's something I introduced in a recent PR, here's how it looks!
https://github.com/snyk/kubernetes-monitor/blob/staging/.circleci/config.yml#L32-L44

Reply:
amazing thanks! :-)